src/Action/CompareAction.php

```diff
@@ -37,10 +37,12 @@ public function __invoke(Request $request, string $className, string $id, ?int $
     {
         if (null === $oldRev) {
             $oldRev = $request->query->get('oldRev');
+            \assert(null !== $oldRev);
         }
 
         if (null === $newRev) {
             $newRev = $request->query->get('newRev');
+            \assert(null !== $newRev);
         }
```

Comment:
this code is a bit strange isnt it?

Reply:
Yes.

The route is
```
simple_things_entity_audit_compare:
    path: /compare/{className}/{id}/{oldRev}/{newRev}
    controller: SimpleThings\EntityAudit\Action\CompareAction
    defaults:
        oldRev: ~
        newRev: ~
```
and the controller
```
    public function compareAction(Request $request, $className, $id, $oldRev = null, $newRev = null)
    {
        $compareAction = new CompareAction($this->get('twig'), $this->getAuditReader());

        return $compareAction($request, $className, $id, $oldRev, $newRev);
    }
```

Reply:
Well if you are going to compare things, better pasa the things to compare right? I guess we should remove null yes.

Reply:
Sure but remove it so far is a BC break.

Also, I don't understand why it would be in 
```
$request->query->get('oldRev');
```
and not passed by the controller, since the routeParam is `oldRev` too...

Seems like it's coming from https://github.com/sonata-project/EntityAuditBundle/blob/1.x/src/Resources/views/Audit/view_entity.html.twig#L31-L32
Since I wasn't sure about what I could do, I prefer to avoid breaking things